authentication/pipeline/user.py

```diff
@@ -20,7 +20,7 @@
 from profiles.utils import update_full_name
 
 
-def validate_email_auth_request(strategy, backend, user=None, *args, **kwargs):  # pylint: disable=unused-argument
+def validate_email_auth_request(strategy, backend, user=None, **kwargs):  # pylint: disable=unused-argument
```

Comment:
We should still keep these around just in case there is an extra positional argument and we don't want to error on that. But we can move `*args` before `user=None` to fix the warning

Reply:
Moving `*args` before `user` causes a test to fail (`auth.pipeline.user_test.test_get_username`).  I think it might be best to just leave these functions in their original state and add an ignore for `keyword-arg-before-vararg` to the top, so as to avoid any nasty surprises with social-auth.

Reply:
I reverted the code for `auth.pipeline.user` to be the same as master and added `# pylint: disable=keyword-arg-before-vararg` to the top of the file, I think this is safer for now.